mlflow/tracking/fluent.py

```diff
@@ -1125,7 +1128,9 @@ def search_runs(
            metrics.m tags.s.release                            run_id
         0       1.55       1.1.0-RC  5cc7feaf532f496f885ad7750809c4d4
     """
-    if not experiment_ids:
+    if search_all_experiments and (experiment_ids is None or len(experiment_ids) == 0):
+        experiment_ids = [exp.experiment_id for exp in list_experiments(view_type=ViewType.ALL)]
```

Comment:
What's the reason to set the view_type to ALL instead of ACTIVE_ONLY?

Reply:
I'm not tied to view_type=ALL, was just working with the broadest possible search for this convenience feature. Happy to change it - either way, the customer can always get exactly the search they want by getting their own list of experiment_ids first

Reply:
I'm not opinionated so whatever you think is best!

Reply:
Done!